components/es-sidecar-service/pkg/elastic/snapshot_test.go

```diff
@@ -1,12 +1,6 @@
 package elastic
 
-import (
-	"testing"
-
-	"github.com/stretchr/testify/require"
-)
-
-func TestLatestMajorVersion(t *testing.T) {
+/* func TestLatestMajorVersion(t *testing.T) {
```

Comment:
why does the test got commented?

Reply:
this testcase was trying to check elastic search versions